bundles/org.openhab.binding.automower/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -164,4 +181,77 @@
 		<state readOnly="true"/>
 	</channel-type>
 
+	<channel-type id="plannerNextStartTimestampType">
+		<item-type>DateTime</item-type>
+		<label>The time for the next auto start</label>
```

Comment:
Labels should be concise as possible. Also, words in labels should be capitalized (except prepositions and so on). See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions
```suggestion
		<label>Next Auto Start</label>
```